src/resolvers-cassandra/Settings/mutations.js

```diff
@@ -63,6 +65,18 @@ function modifyBlacklist(args, res) { // eslint-disable-line no-unused-vars
  * @returns {Promise.<{runTime: string, filters: Array<{filteredTerms: string[], lang: string, RowKey: string}>}>}
  */
 function removeBlacklist(args, res) { // eslint-disable-line no-unused-vars
+  return new Promise((resolve, reject) => {
+    if (!args.input) return reject('No input specified');
+    if (!args.input.terms) return reject('No terms specified');
+    if (!args.input.terms.length) return reject('No terms specified');
+    const terms = args.input.terms;
```

Comment:
Can inline, especially since we're dereferencing the args.input.terms again [below](https://github.com/CatalystCode/project-fortis-services/pull/53/files#diff-81c1265e293bf42b95e265ce1401528bR76).